src/io/flutter/view/FlutterView.java

```diff
@@ -201,58 +196,12 @@ private void addInspectorViewContent(FlutterApp app, @Nullable InspectorService
       tabName = device.getUniqueName(existingDevices);
     }
 
-    final Content content = contentManager.getFactory().createContent(null, tabName, false);
-    tabContainer.add(runnerTabs.getComponent(), BorderLayout.CENTER);
-    content.setComponent(tabContainer);
-    content.putUserData(ToolWindow.SHOW_CONTENT_ICON, Boolean.TRUE);
-    content.setIcon(FlutterIcons.Phone);
-    contentManager.addContent(content);
+    DevToolsManager.getInstance(app.getProject()).openBrowserIntoPanel(browserUrl, contentManager, tabName);
```

Comment:
How confident are we with the process here (downloading, jsbrowser, ...)? We should probably do something like have an experimental setting in our preferences, and let people opt into trying this out for release or two. That would allow us to iron out any issues.

Reply:
We're going to start with these features disabled, since there are some other pieces that would be too much to add in this change. I think it would make sense to add a setting to opt in/out, especially since we will need a permanent way to opt out.